### ISSUE TITLE
Retraining model without nawar.yaml file

### ISSUE THREAD

**Issue by ZeyadAyman1** [nipponjo/tts-arabic-pytorch#19]
Hello, the training script and tutorial mentions using  a config file called nawar.yaml , i wanted to ask why it was removed from the repo as if i wanted to download and use it will i face any issues? also is the train.py script updated  to reflect the absence of that file as i tried different config files from the present files and i find that there are some variables that needs to be removed and such . 
Thanks.

**Comment by nipponjo** [comment]
Hi,
initially this repo only contained the Tacotron model with its default loss function; so therefore I only had one training script and config file. Later, I added an adversarial loss and the FastPitch model. The repo has different training scripts (located in the `/scripts` folder) for each of these 4 combinations: 

`train_tc.py`: Tacotron without adversarial loss
`train_tc_adv.py`: Tacotron with adversarial loss
`train_fp.py`: FastPitch without adversarial loss
`train_fp_adv.py`: FastPitch with adversarial loss

I have put these into the `/scripts` folder in order to not have so many files in the root folder. The config files use the corresponding suffixes (_fp_adv, ...). I recommend using the _adv files, since training is generally stable in my experience, and will probably merge both versions into one file in the future.

**Comment by ZeyadAyman1** [comment]
Thanks for the fast response, i tried using the train_fp_adv.py but i got an error saying that "./data/pitch_dict.pt" is not found .
